### DISCUSSION TITLE
transcending the limits of crypto.

### DISCUSSION THREAD

**Discussion by metinx33** [IBAX-io/go-ibax#1151] [Ideas] [answered]
### Clear and concise description of the problem

think about it... A simple and universal wallet that can be easily used on shopping sites and regular payments.
To be able to make our payments easily with the digital assets we have, with a knowing wallet that can make instant, fast and easy exchange and conversion by offering attractive offers to famous shopping sites.
Is it a dream...
that would be the end of fiat money.

### Suggested solution

To design a universal and multi-purpose wallet that quickly converts the digital assets we have to the required amount while shopping online or making any payments, and at the same time displays the amount we need to pay on the basis of the digital assets we own.

### Alternative

IBAX UNIVERSAL WALLET

(Amazan.com)Amount Payable : $250

     -ASSETS OWNED-     -CURRENT VALUE-          -PROVISION FOR AMOUNT PAYABLE-
IBXC : 1250000....................IBXC : 57,3500$.....................= 4,3591.IBXC.................[PAY WITH IBXC]         
BTC : 1,2500..........................BTC : 36448,5700$...............= 0,0068.BTC...................[PAY WITH BTC] 
ETH : 24,7300.......................ETH : 2435,1600$.................= 0,1026.ETH...................[PAY WITH ETH]
SHIB : 15000000..................SHIB : 0,00002069$............= 12083,1319.SHIB.......[PAY WITH SHIB] 
DOGE : 300000......................DOGE : 0,1424$....................= 1755,6179.DOGE.......[PAY WITH DOGE]
ATOM : 265............................ATOM : 31,5200$................= 7,9314.ATOM................[PAY WITH ATOM]

### Additional context

An idea is like a snowflake that will start an avalanche.
It should never be taken lightly.

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by HalloKyle** [comment, marked as answer]
This is a great idea and you can create an ecosystem in IBAX to implement this business scenario. If it is online shopping we can tell that you may encounter several critical issues.
1. the identity information of the shopper and seller must be able to be identified. (according to your needs)
2. logistics information can be tracked on the chain and can be traced (and the privacy of both sides of the transaction needs to be protected).
3. support for multiple cryptocurrency payments, preferably using Defi direct transactions.

We are building an on-chain DID identity verification function that allows both sides of a transaction to complete identity verification and confirmation (you can also build your own DID verification model.) IBAX's CLB has cross-chain communication capabilities and supports data and assets across chains, so you can use her to complete the second and third points of functionality. Using CLB you can access the content of existing established logistics providers and link their logistics information to purchase orders, which is data that only the transacting party can view or authorize to allow to be made public.

We encourages you to implement your own ideas.

**Comment by metinx33** [comment]
Thank you so much kyle for your answer.
Sorry, I'm not a programmer and I don't even know what the terms like 'DID', 'CLB' you mentioned above mean.
I recognized IBAX as a platform where people can come up with ideas, even if they do not have any software knowledge, and I will act as such.
I would love to know software, but I have no knowledge about it, I am just an idea person.
Look, another idea came to my mind...
-Creating crypto virtual post system such as VISA and MASTER CARD.
-Providing integrated virtual post service with the application of shopping or other payment systems.
-Receiving payment when IBAX WALLET or any wallet is connected to the integrated crypto virtual post system.
-To be able to return the amount received to the registered wallet address that made the payment during the return process of the purchased product or service.[
![image](https://user-images.githubusercontent.com/98493027/151429571-31cd8b71-11ef-48ea-b101-4e5607219af1.png)
](url)

**Comment by CannyCrow** [comment]
Crypto Wallet
